problemsets-mat223/linearalbegra.tex

```diff
@@ -2111,8 +2111,20 @@ \section*{Dot Product}
 		\item Use set-builder notation to write down the set, $X$, of
 			all vectors orthogonal to $\vec n$. Describe this set
 			geometrically.
+			\begin{solution}
+				$X = \Set*{\vec x\in \R^2 \given \vec x\cdot \vec n = 0}$.
+
+				Geometrically, this is a line through the origin and 
```

Comment:
It's a plane

Reply:
Whoops. Sorry about that. Wrote \R^2 by accident and then ran with that I guess. Fixing now.